packages/skills/carpark_client/handlers.py

```diff
@@ -81,22 +80,22 @@ def handle(self, message: Message, sender: str) -> None:
 
         # recover dialogue
         dialogues = cast(Dialogues, self.context.dialogues)
-        if dialogues.is_belonging_to_registered_dialogue(fipa_msg, sender, self.context.agent_public_key):
-            dialogue = cast(Dialogue, dialogues.get_dialogue(fipa_msg, sender, self.context.agent_public_key))
+        if dialogues.is_belonging_to_registered_dialogue(fipa_msg, self.context.agent_public_key):
+            dialogue = cast(Dialogue, dialogues.get_dialogue(fipa_msg, self.context.agent_public_key))
             dialogue.incoming_extend(fipa_msg)
         else:
-            self._handle_unidentified_dialogue(fipa_msg, sender)
+            self._handle_unidentified_dialogue(fipa_msg)
             return
 
         # handle message
         if msg_performative == FIPAMessage.Performative.PROPOSE:
-            self._handle_propose(fipa_msg, sender, message_id, dialogue)
+            self._handle_propose(fipa_msg, message_id, dialogue)
```

Comment:
Not sure why message_id is passed in, that can be recovered from fipa_msg, so please change that too

Reply:
I can see that there is a message_id inside the FIPAMessage class, but when I am using `msg.message_id +1` gives me -> Unresolved attribute reference 'mesasge_id' for class 'FIPAMessage'.